test/WorkerHandler.test.js

```diff
@@ -183,7 +183,7 @@ describe('WorkerHandler', function () {
         });
   });
 
-  it('should cancel a task', function () {
+  it.only('should cancel a task', function () {
```

Comment:
Can you remove the `.only` from this test again? 😉

Reply:
lol